scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/executors/forked/env/ExecuteForkedTaskInsideNewJvm.java

```diff
@@ -43,17 +43,42 @@
 
 public class ExecuteForkedTaskInsideNewJvm {
 
+    public static final int MAX_CONTEXT_WAIT = 6000;
+
+    public static final String CONTEXT_FILE = "Context file ";
+
     // 2 called by forked
     private static TaskContext deserializeContext(String pathToFile) throws IOException,
```

Comment:
check if is possible to remove the static private methods and have a simple test for this class